src/test/java/com/diyasylum/formfiller/FillerCliTest.java

```diff
@@ -0,0 +1,63 @@
+package com.diyasylum.formfiller;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import com.diyasylum.formfiller.models.FieldType;
+import com.diyasylum.formfiller.models.I589Field;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.Arrays;
+import java.util.List;
+import org.apache.pdfbox.pdmodel.PDDocument;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junitpioneer.jupiter.TempDirectory;
+import picocli.CommandLine;
+
+class FillerCliTest {
+
+  @Test
+  @ExtendWith(TempDirectory.class)
+  void run(@TempDirectory.TempDir Path tempDir) throws IOException {
+    Path fields = tempDir.resolve("fields.json");
+    FileWriter fileWriter = new FileWriter(fields.toFile());
+    ObjectMapper objectMapper = new ObjectMapper();
+    List<I589Field> filledIn =
```

Comment:
Tests! Nice. One random test to add: check to make sure that the "/" in "N/A" actually renders in all the fields we need it to. I remember Jamie walked me through the form once, and the form itself did not allow you to type non-alphanumeric characters in certain fields, despite the instructions clearly saying to put "N/A" in some of them.

Reply:
innnnnnnnnnnteresting. I may test that one by hand quickly using the new tool.

Reply:
https://drive.google.com/file/d/15YFu-Zw2N-BafUrzohDiWSLONLInh9eH/view?usp=sharing